db/migrate/20210426193009_move_approved_queued_posts_topic_and_post_data.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class MoveApprovedQueuedPostsTopicAndPostData < ActiveRecord::Migration[6.1]
+  def up
+    DB.query_single <<~SQL
+      UPDATE reviewables r
+      SET topic_id = (payload->>'created_topic_id')::int, target_id = (payload->>'created_post_id')::int
+      WHERE r.type = 'ReviewableQueuedPost' AND r.status = #{Reviewable.statuses[:approved]}
```

Comment:
I would avoid reaching into active record here and just hard code the status, makes the migration less brittle.

Reply:
maybe also for extra safty look for `topic_id IS NULL`  and `target_id IS NULL` ? Maybe to limit the risks here we just forgo the migration altogether? on the fence on that but the migration parses and casts so there is an element of risk.